apps/server/src/modules/tool/context-external-tool/controller/api-test/tool-context.api.spec.ts

```diff
@@ -245,41 +212,42 @@ describe('ToolContextController (API)', () => {
 					toolVersion: 1,
 				});
 
-				em.persist([course, userWithMissingPermission, schoolExternalToolEntity, contextExternalToolEntity]);
+				em.persist([course, teacherUser, teacherAccount, schoolExternalToolEntity, contextExternalToolEntity]);
 				await em.flush();
 				em.clear();
 
+				const loggedInClient: TestApiClient = await testApiClient.login(teacherAccount);
+
 				return {
 					contextExternalToolEntity,
-					userWithMissingPermission,
+					loggedInClient,
 				};
 			};
 
 			it('should return forbidden when user is not authorized', async () => {
-				const { userWithMissingPermission, contextExternalToolEntity } = await setup();
-				currentUser = mapUserToCurrentUser(userWithMissingPermission);
+				const { loggedInClient, contextExternalToolEntity } = await setup();
+
+				const result = await loggedInClient.delete(`${contextExternalToolEntity.id}`);
 
-				await request(app.getHttpServer()).delete(`${basePath}/${contextExternalToolEntity.id}`).expect(403);
+				expect(result.statusCode).toEqual(403);
```

Comment:
Use here and in api spec s which touched please HttpStatus.NO_CONTET, HttpStatus.OK.. and so on. That's better than just use just the statuscodes as number.
HttpStatus comes from nestjs